settings/src/client.rs

```diff
@@ -40,6 +40,11 @@ pub struct ExitServer {
     #[serde(default)]
     pub registration_port: u16,
 
+    /// The power we reach out to to hit the register endpoint
+    /// also used for all other exit lifecycle management api calls
+    #[serde(default)]
+    pub wg_exit_listen_port: u16,
```

Comment:
this is going to default to zero, please use the default function feature to specify the correct default value.

Reply:
added 4875 default registration port and 59998 default wg exit listen port